instrumentation/aws-lambda/aws-lambda-core-1.0/testing/src/main/java/io/opentelemetry/instrumentation/awslambdacore/v1_0/AbstractAwsLambdaTest.java

```diff
@@ -90,39 +87,4 @@ void handlerTracedWithException() {
                             .hasAttributesSatisfyingExactly(
                                 equalTo(SemanticAttributes.FAAS_INVOCATION_ID, "1-22-333"))));
   }
-
-  @Test
-  @SetEnvironmentVariable(
-      key = "_X_AMZN_TRACE_ID",
-      value = "Root=1-8a3c60f7-d188f8fa79d48a391a778fa6;Parent=0000000000000456;Sampled=1")
-  void handlerLinksToInfrastructureTrace() {
-    String result = handler().handleRequest("hello", context);
-    assertThat(result).isEqualTo("world");
-
-    testing()
-        .waitAndAssertTraces(
-            trace ->
-                trace.hasSpansSatisfyingExactly(
-                    span ->
-                        span.hasName("my_function")
-                            .hasKind(SpanKind.SERVER)
-                            .hasLinksSatisfying(
```

Comment:
would it make sense to keep rest of the test except the `hasLinksSatisfying` part?

Reply:
I thought I added this test when I added the span link. It was actually added before, so I've updated the test instead now.